website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -363,11 +377,11 @@ resource "kubectl_manifest" "karpenter_provisioner" {
         cpu: 1000
     provider:
       subnetSelector:
-        karpenter.sh/discovery: ${local.cluster_name}
+        Name: "*private*"
       securityGroupSelector:
-        karpenter.sh/discovery: ${local.cluster_name}
+        karpenter.sh/discovery/${module.eks.cluster_id}: ${module.eks.cluster_id}
```

Comment:
The cluster unique key is directly related to #1488 - this allows users to have multiple clusters in a single account without tag conflicts and still proper permissions.